src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -262,6 +264,32 @@ private void execute() {
                     addCustomValueAndSearchLink(buildScan, "CI job", value));
                 envVariable("CI_JOB_STAGE").ifPresent(value ->
                     addCustomValueAndSearchLink(buildScan, "CI stage", value));
+                // 'git' might not be installed in gitlab docker runners, use the equivalent environment variables
+                commitSha.ifPresent(gitCommitId ->
+                        buildScan.value("Git commit id", gitCommitId));
+                envVariable("CI_COMMIT_SHORT_SHA").ifPresent(gitCommitShortId ->
+                        addCustomValueAndSearchLink(buildScan,
+                            "Git commit id",
+                            "Git commit id short",
+                            gitCommitShortId));
+                envVariable("CI_REPOSITORY_URL").ifPresent(gitRepo ->
```

Comment:
This could instead by the project URL, which is the browser URL, `CI_REPOSITORY_URL` is the git remote URL)
```suggestion
                projectUrl.ifPresent(gitRepo ->
```